config/src/config/consensus_config.rs

```diff
@@ -48,10 +49,10 @@ pub struct ChainHealthBackoffValues {
 impl Default for ConsensusConfig {
     fn default() -> ConsensusConfig {
         ConsensusConfig {
-            max_sending_block_txns: 2500,
+            max_sending_block_txns: 4000,
```

Comment:
@danielxiangzl @sasha8 do we want these consensus changes when quorum store is enabled? @igor-aptos would it make sense to make these changes without quorum store at all?

Reply:
I think we need larger block size to fully utilize the potential of QS. We can tune these parameters after landing in testnet/production.